neo/test/test_utils.py

```diff
@@ -20,7 +20,10 @@
                             assert_same_attributes,
                             assert_same_annotations)
 
-from neo.utils import (get_events, get_epochs, add_epoch, match_events, cut_block_by_epochs)
+from neo.utils import (get_events, get_epochs, add_epoch, match_events,
```

Comment:
I don't think this change is needed since Neo allows for up to 99 characters per line (https://github.com/NeuralEnsemble/python-neo/blob/57f8d079deef7533394b4c1e0db103bf963d203c/.pep8speaks.yml#L2) Maybe you are using the default PEP8 settings for autoformatting here?